data_model/src/lib.rs

```diff
@@ -53,6 +53,8 @@ use crate::{account::SignatureCheckCondition, name::Name};
 pub mod account;
 pub mod asset;
 pub mod block;
+#[doc(hidden)]
+pub mod derive_private;
```

Comment:
why does it have to be `pub`?

Reply:
The idea is to have a stable names to refer to in a derive macro imports. This is often used for simplicity and whatnot, but in our case it was useful to resolve `Cow` to a correct name both in case of `no_std` and `std` configurations. There's no single import path that works for both, and you can't use `#[cfg]` in generated code, because you can't expect the using crate to have specific feature names. This pattern ensures that the `#[cfg]` is evaluated in a known crate with known feature names.

But I actually removed the usage of `Cow` (because it actually didn't do what I needed it to), so this should go now.